tools/ci_build/github/azure-pipelines/linux-gpu-ci-pipeline-cuda9.yml

```diff
@@ -0,0 +1,12 @@
+jobs:
+- job: Linux_CI_GPU_Dev
+  pool: Linux-GPU-CUDA10
```

Comment:
why is the pool name cuda10 when the cuda version used is 9.1? do we've just one pool?

Reply:
this is the build agent pool name. We created separate pool to make the switch smooth

---
In reply to: [239301328](https://github.com/Microsoft/onnxruntime/pull/112#discussion_r239301328) [](ancestors = 239301328)